### ISSUE TITLE
Mindmap feature not open source

### ISSUE THREAD

**Issue by howardroark2018** [leanote/leanote#414]
hi,
I am a fan of the software and I use it daily. Recently I had been experimenting with the code and I couldn't find the mindmap functionality of the software similar to leanote.com in the source code on Github. Is Mindmap functionality not released under open source license?

Thanks a lot for the beautiful work you guys have been doing! :)

**Comment by ystyle** [comment]
You can try the [processon](https://www.processon.com)

**Comment by lealife** [comment]
Yes, Mindmap functionality not released under open source license

**Comment by lealife** [comment]
It will be released in the furture
